websites/#/アイドルマスターシャイニーカラーズ/presence.ts

```diff
@@ -0,0 +1,91 @@
+const presence = new Presence({
+		clientId: "1103931016525127792",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const { pathname } = document.location,
+		presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/ue8EatG.jpg",
```

Comment:
This image does not seem to exist.
```suggestion
			largeImageKey: "https://i.imgur.com/YfaIAPE.png",
```